tiledb/CMakeLists.txt

```diff
@@ -1088,7 +1091,10 @@ if (TILEDB_STATIC)
     )
     # Create the target
     add_library(tiledb_static STATIC
-      $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS_STATIC>
+      $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS_STATIC>)
```

Comment:
The problem might be here. `STATIC` libraries expect sources, so `add_library` might be treating the target objects as sources, not finding sources, and ignoring them. Adding the same sources in `target_link_libraries` may work.

Reply:
This line existed before. And the problem is that it does not add the objects from `baseline`, not `TILEDB_CORE_OBJECTS_STATIC`.

I also checked the generated Ninja file; the `TILEDB_CORE_OBJECTS_STATIC` target has an order-only dependency to `baseline`, but the objects are not included in the `tiledb_static` target. This does not happen with `TILEDB_CORE_OBJECTS` and `tiledb_shared`.